plugins/xpressive/exprtk_patch

```diff
@@ -0,0 +1,64 @@
+57,58c57,69
```

Comment:
`exprtk.hpp.diff` or `exprtk.diff` would be the accepted file extension.  This will trigger syntax highlighting in most code editors.